test/clj_http/test/core_test.clj

```diff
@@ -99,7 +99,9 @@
     [:propfind "/propfind"]
     {:status 200 :body "propfind"}
     [:propfind "/propfind-with-body"]
-    {:status 200 :body (:body req)}))
+    {:status 200 :body (:body req)}
+    [:get "/query-string"]
+    {:status 200 :body (:query-string req)}))
```

Comment:
This was added but isn't used at all? Maybe this is missing the associated test?